components/tikv_alloc/src/default.rs

```diff
@@ -9,3 +10,11 @@ pub fn dump_prof(_path: Option<&str>) {}
 pub fn fetch_stats() -> io::Result<Option<AllocStats>> {
     Ok(None)
 }
+
+pub fn activate_prof() -> ProfResult<()> {
+    Err(ProfError::JemallocNotEnabled)
```

Comment:
A little strange that the common function just returns a Jemalloc error.

Reply:
Better use `ProfileDisabled`. Actually tcmalloc also support profiling.